<launch>
  
  <rosparam file="$(find data_acquisition)/param/params.yaml" />
  
  <node pkg="data_acquisition" type="encoderData.py" name="encoderSensor" />
  
  <node pkg="data_acquisition" type="monoData.py" name="monoSensor" />
  
  <node pkg="data_acquisition" type="stereoData.py" name="stereoSensor" />
  
  <node pkg="data_acquisition" type="synchronizer.py" name="dual_sync_node" output="screen" />
  
</launch>

